iDiff/differs/fileDiff.go

```diff
@@ -33,20 +32,6 @@ func diffImageFiles(image1, image2 utils.Image) (utils.DirDiff, error) {
 		return diff, fmt.Errorf("Error parsing image %s contents: %s", img2, err)
 	}
 
-	for layer1, contents1 := range img1Contents {
-		sameLayer := false
-		for layer2, contents2 := range img2Contents {
-			if checkSameLayer(contents1, contents2) {
-				delete(img2Contents, layer2)
-				sameLayer = true
-				break
-			}
-		}
-		if sameLayer {
-			delete(img1Contents, layer1)
-		}
-	}
-
 	img1Dir := utils.Directory{
 		Root:    img1,
 		Content: getContentList(img1Contents),
```

Comment:
just turn the image FS into a Directory struct immediately from its path without all this extra content getting stuff